packages/veritone-react-common/src/components/MediaInfoPanel/EditMetadataDialog/index.js

```diff
@@ -128,103 +147,121 @@ class EditMetadataDialog extends Component {
-                    src={this.state.programImage}
-                  />
-                )}
-              {(!this.state.programImage ||
-                !this.state.programImage.length) && (
-                <img
-                  className={styles.programImage}
-                  src="//static.veritone.com/veritone-ui/program_image_null.svg"
-                />
-              )}
-            </div>
-          </div>
-        </DialogContent>
-        <DialogActions>
-          <Button onClick={this.props.onClose} color="primary">
-            Cancel
-          </Button>
-          <Button
-            onClick={this.onSave}
-            color="primary"
-            disabled={!this.hasPendingChanges()}
-          >
-            Save
-          </Button>
-        </DialogActions>
+        )}
+        {!this.state.selectingFile && (
+          <Fragment>
+            <DialogTitle
+              classes={{
+                root: styles.dialogTitle
+              }}
+            >
+              <div className={styles.dialogTitleLabel}>Edit Metadata</div>
+              <IconButton
+                onClick={this.props.onClose}
+                aria-label="Close"
+                classes={{
+                  root: styles.closeButton
+                }}
+              >
+                <Icon className="icon-close-exit" />
+              </IconButton>
+            </DialogTitle>
+            <DialogContent>
+              <TextField
+                autoFocus
+                margin="normal"
+                id="filename"
+                label="Filename"
+                InputLabelProps={{
+                  shrink: true
+                }}
+                value={this.state.filename}
+                onChange={this.onFileNameChange}
+                fullWidth
+                classes={{
+                  root: styles.textInput
+                }}
+              />
+              <TextField
+                margin="normal"
+                id="source"
+                className={styles.sourceSection}
+                label="Source"
+                placeholder="Enter name of media owner or creator"
+                InputLabelProps={{
+                  shrink: true
+                }}
+                value={this.state.source}
+                onChange={this.onSourceChange}
+                fullWidth
+                classes={{
+                  root: styles.textInput
+                }}
+              />
+              <div className={styles.programImagesSection}>
+                <div
+                  className={styles.imageSection}
+                >
+                  <div className={styles.programImageLabel}>
+                    Program Live Image
+                  </div>
+                  <div className={styles.programLiveImageContainer}>
+                    <img className={styles.programLiveImage}
+                      src={
+                        get(this.state, 'programLiveImage.length') ?
```

Comment:
is there any reason you can't do -> this.state.programLiveImage || '//static.veritone.com/veritone-ui/default-nullstate.svg'?